apps/src/code-studio/pd/application_dashboard/quick_view.jsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * Application Dashboard quick view.
+ * Route: /csd_teachers
+ *        /csp_teachers
+ *        /csd_facilitators
+ *        /csp_facilitators
+ */
+import React, {PropTypes} from 'react';
+import QuickViewTable from './quick_view_table';
+
+export default class QuickView extends React.Component {
+  static propTypes = {
+    route: PropTypes.shape({
+      title: PropTypes.string.isRequired
+    })
+  };
+
+  render() {
+    return (
+      <div>
+        <h1>{this.props.route.title}</h1>
+        <QuickViewTable />
```

Comment:
So these are all the same at the moment? 

That's fine for the initial PR, but we'll eventually need to pass a prop specifying which type of applications to load and display.

Reply:
Ideally, I'd like to see a little more of this wiring here. No need to build unnecessary scaffolds, or supply real data yet, but it would be nice if the various controls at least had their necessary context.

Reply:
Additional props will be coming in a follow up PR.